interactive/admin.py

```diff
@@ -17,8 +19,46 @@ class UserAdmin(admin.ModelAdmin):
 # admin.site.unregister(auth.models.User)
 admin.site.unregister(auth.models.Group)
 
-# Register your models to admin site, then you can add, edit, delete and search your models in Django admin site.
-admin.site.register(RegistrationRequest)
+
+@admin.register(RegistrationRequest)
+class RegistrationRequestAdmin(admin.ModelAdmin):
+    change_form_template = "admin/registration_review_form.html"
+    list_display = ["full_name", "email", "job_title", "organisation", "review_status"]
+
+    def get_readonly_fields(self, request, obj=None):
+        if obj:
+            return [
+                "full_name",
+                "email",
+                "job_title",
+                "organisation",
+                "review_status",
+                "reviewed_at",
+                "reviewed_by",
+            ]
+        else:
+            return []  # pragma: no cover
```

Comment:
If you flip the conditional here you can short cut the check, eg:
```
if not obj:
    return []

return [
    ...
]
```

Saves the reader having to find the dangling else 🙂

Reply:
Good point. Thank you!